geospatial/content/templates/content/event_page.html

```diff
@@ -0,0 +1,26 @@
+{% extends "base.html" %}
+{% load wagtailcore_tags %}
+
+{% block body_class %}template-EventPage{% endblock %}
+
+{% block content %}
+
+  <h1>{{ page.title }}</h1>
+
+  <ul>
+    {% for author in page.authors.all %}
+    <li>
+      <a href="{{ author.url }}">
+        {{ author.title }}
+      </a>
+    </li>
+    {% endfor %}
+  </ul>
+  
+  <p>{{ page.date|safe }}</p>
```

Comment:
The date shouldn't need a `|safe` filter since it's not raw HTML. It's best to avoid the filter unless it's explicitly necessary because it's always a vector for [XSS attacks](https://docs.djangoproject.com/en/3.2/topics/security/#cross-site-scripting-xss-protection).

We should, however, define a uniform representation of dates and times, across the site. In templates, [the `|date` filter](https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#date) should help, along with the [`DATE_FORMAT` setting](https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-DATE_FORMAT) and its friends (`DATETIME_FORMAT`, `TIME_FORMAT` and `SHORT_DATE_FORMAT`). We should set them in `settings.py` to some version of a Romanian standard format. Which reminds me, we should probably set [`LANGUAGE_CODE`](https://docs.djangoproject.com/en/3.2/ref/settings/#std:setting-LANGUAGE_CODE) to `ro`.